### PR TITLE
brakeman and bullet gem installed

### PR DESCRIPTION
**Pull request by 25shwetak**

### CODE REVIEW

**Comment by ksnehanjali** [comment]
[Sign in page with validations](https://trello.com/c/UWirGN3Z/48-sign-in-page-with-validations)

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,8 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'bullet'
+  gem 'brakeman'
```
Comment (tanya-saroha) [inline Gemfile]: @25shwetak Whenever you are adding any new gems it is a good practice to add a short comment with it stating why you have added that gem.

File: config/environments/development.rb [hunk 2]
```diff
@@ -51,4 +51,11 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.after_initialize do
+    Bullet.enable = true
+    Bullrt.alert = ENV['BULLET_ALERTS']
```
Comment (tanya-saroha) [inline config/environments/development.rb]: @25shwetak Please correct the spelling here

Comment (tanya-saroha) [inline config/environments/development.rb]: @25shwetak Why have you user ENV['BULLET_ALERTS'] here?

File: config/environments/development.rb [hunk 3]
```diff
@@ -51,4 +51,11 @@
...
+    Bullet.bullet_logger = true
```
Comment (tanya-saroha) [inline config/environments/development.rb]: @25shwetak Please correct the spelling here

File: config/environments/development.rb (hunk 2 above)
Comment (25shwetak) [reply config/environments/development.rb]: Ma'am I have changed it to 'Bullet.alert = true'.

**Comment by tanya-saroha** [comment]
@25shwetak I see the same commit message "changes resolved" and different changes inside. Your commit messages should  be in written in a way that anybody who is reading the commit message should know what changes are made in that commit just by reading. Take care from next time.

File: config/environments/development.rb (hunk 2 above)
Comment (tanya-saroha) [reply config/environments/development.rb]: Dont use ma'am

**Pull request merged**